Kernel/Storage/AHCIPort.cpp

```diff
@@ -117,7 +117,7 @@ void AHCIPort::handle_interrupt()
         if (!m_current_request) {
             dbgln_if(AHCI_DEBUG, "AHCI Port {}: Request handled, probably identify request", representative_port_index());
         } else {
-            g_io_work->queue([this]() {
+            g_io_work->queue([this](void*) {
```

Comment:
This should not be needed, if we use lambdas we should just capture what we need and there should be no need for arguments unless you pass a static free function to it.